mypy/fastparse2.py

```diff
@@ -144,7 +141,10 @@ def generic_visit(self, node: ast27.AST) -> None:
     def visit_NoneType(self, n: Any) -> Optional[Node]:
         return None
 
-    def visit_list(self, l: Sequence[ast27.AST]) -> List[Node]:
+    def visit_list_expression(self, l: Sequence[ast27.AST]) -> List[Expression]:
```

Comment:
I'd expect this to be named visit_expression_list.

FWIW I wish there weren't helper methods named `visit_*`, since that name pattern is also used by the dynamic dispatch. So maybe both this and visit_statement_list should be renamed.